sdk/src/main/java/io/radar/sdk/Radar.kt

```diff
@@ -3077,6 +3082,15 @@ object Radar {
         replayBuffer.write(replayParams)
     }
 
+    @JvmStatic
+    internal fun loadReplayBufferFromSharedPreferences() {
+        replayBuffer.loadFromSharedPreferences()
+        val replays = getReplays()
+        val replayCount = replays.size 
+        // TODO: revisit this log and log type
+        logger.i("loadReplayBufferFromSharedPreferences() replayCount = $replayCount", RadarLogType.SDK_CALL)
```

Comment:
can keep as a debug level log if we want the visibility (vs `info`)
maybe "loaded replay buffer from shared preferences with X replays"